plugins/USBPrinting/USBPrinterOutputDevice.py

```diff
@@ -182,7 +183,7 @@ def _updateFirmware(self):
 
         if len(hex_file) == 0:
             Logger.log("e", "Unable to read provided hex file. Could not update firmware")
-            self._updateFirmware_completed()
+            self._updateFirmware_completed_missing_firmware()
```

Comment:
Sorry for not catching this before. The codestyle for private method names in python is _lowerCamelCase().